doc/md/fundamentals/2-types/10-options-results.md

```diff
@@ -139,17 +143,13 @@ If either `firstName` or `lastName` is `null`, the result remains `null`.
 ### Unwrapping an option
```

Comment:
This section is probably redundant now. Also its inconsistent since the example use let-else, but the text is explaining switch.